crates/modelardb_server/src/storage/mod.rs

```diff
@@ -424,7 +425,7 @@ impl StorageEngine {
     pub(super) async fn update_remote_data_folder(
         &mut self,
         remote_data_folder: Arc<dyn ObjectStore>,
-        metadata_manager: &Arc<MetadataManager>,
+        metadata_manager: &Arc<TableMetadataManager<Sqlite>>,
```

Comment:
Maybe renamed to `table_metadata_manager` for consistency.

Reply:
Fixed.